plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/StoreConnect.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+
+/**
+ * Connect store to Woo.com Task
+ */
+class StoreConnect extends Task {
+	/**
+	 * ID.
+	 *
+	 * @return string
+	 */
+	public function get_id() {
+		return 'connect-store';
+	}
+
+	/**
+	 * Title.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		return __( 'Connect store to Woo.com', 'woocommerce' );
```

Comment:
Could we change the copy to something like `Manage your Woo Marketplace subscriptions"? This is a similar style to the latest copy in the main task list. The aim is to describe the benefit to the user and why they should complete the task.

Reply:
Addressed [30f8b1e](https://github.com/woocommerce/woocommerce/pull/45795/commits/30f8b1ebff0ac8b251a449dd1a3ea0c793fa87f0)

Thanks @adrianduffell for your suggestion